server/@types/bapv.d.ts

```diff
@@ -134,12 +134,13 @@ export type VisitSessionData = {
     location: string
   }
   visit?: VisitSlot
+  visitRestriction?: 'OPEN' | 'CLOSED'
   visitors?: VisitorListItem[]
   visitorSupport?: VisitorSupport[]
   mainContact?: {
     contact?: VisitorListItem
     phoneNumber: string
     contactName?: string
   }
-  visitId?: string
+  visitReference?: string
```

Comment:
Renamed for consistency with API terminology - and Richard was quite keen on calling it a 'reference' rather than an 'id'.